src/d3a/d3a_core/cli.py

```diff
@@ -19,8 +19,7 @@
 from logging import getLogger
 
 import click
-import dill
```

Comment:
Now that you removed all usages of `dill` you could remove it from the requirements.

Reply:
Will do so , thanks !